2015-11-21 19:36:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAYERS" not found; SQL statement:
SELECT * FROM players WHERE name = ? AND key = ? [42102-188]
2015-11-21 19:42:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "0" for parameter "columnIndex" [90008-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3172)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3200)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:321)
	at com.ormus.server.model.dao.BaseDAO.checkIfExists(BaseDAO.java:37)
	at com.ormus.server.Server.<init>(Server.java:45)
	at com.ormus.server.Server.main(Server.java:88)
2015-11-21 19:42:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "0" for parameter "columnIndex" [90008-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3172)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3200)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:321)
	at com.ormus.server.model.dao.BaseDAO.checkIfExists(BaseDAO.java:37)
	at com.ormus.server.Server.<init>(Server.java:45)
	at com.ormus.server.Server.main(Server.java:88)
2015-11-21 19:42:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "0" for parameter "columnIndex" [90008-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3172)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3200)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:321)
	at com.ormus.server.model.dao.BaseDAO.checkIfExists(BaseDAO.java:37)
	at com.ormus.server.Server.<init>(Server.java:45)
	at com.ormus.server.Server.main(Server.java:88)
2015-11-23 14:26:20 jdbc[10]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.savePlayer(PlayerDAO.java:120)
	at com.ormus.server.entity.World.update(World.java:529)
	at com.ormus.server.entity.World.run(World.java:94)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 14:27:08 jdbc[11]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.getPlayer(PlayerDAO.java:19)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:69)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:93)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 14:59:25 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/ORM/eclipse/workspace/Ormus Server/ormus.mv.db [1.4.188/7]" [50000-188]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2434)
	at org.h2.engine.Database.open(Database.java:671)
	at org.h2.engine.Database.openDatabase(Database.java:268)
	at org.h2.engine.Database.<init>(Database.java:262)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.BaseDAO.checkIfExists(BaseDAO.java:44)
	at com.ormus.server.Server.<init>(Server.java:45)
	at com.ormus.server.Server.main(Server.java:87)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/ORM/eclipse/workspace/Ormus Server/ormus.mv.db [1.4.188/7]" [50000-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/ORM/eclipse/workspace/Ormus Server/ormus.mv.db [1.4.188/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 22 more
2015-11-23 15:01:23 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:508)
	at com.ormus.server.entity.World.run(World.java:94)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-23 15:07:27 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.getPlayer(PlayerDAO.java:19)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:69)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:14:52 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:16:30 jdbc[59]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:18:30 jdbc[24]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:19:24 jdbc[21]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.getPlayer(PlayerDAO.java:19)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:69)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:20:57 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:21:50 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:26:04 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:27:17 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:29:19 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:37:43 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:40:37 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.getPlayer(PlayerDAO.java:19)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:69)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:46:01 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:46:46 jdbc[15]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 15:50:20 jdbc[17]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 16:01:05 jdbc[11]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-23 16:02:24 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:44)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 11:13:21 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.getPlayer(PlayerDAO.java:19)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:69)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 11:17:01 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:509)
	at com.ormus.server.entity.World.run(World.java:94)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-24 11:19:45 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:510)
	at com.ormus.server.entity.World.run(World.java:94)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-24 11:21:37 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:510)
	at com.ormus.server.entity.World.run(World.java:94)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-24 11:24:19 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:508)
	at com.ormus.server.entity.World.run(World.java:94)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-24 11:26:32 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:508)
	at com.ormus.server.entity.World.run(World.java:94)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-24 11:34:09 jdbc[9]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.getPlayer(PlayerDAO.java:19)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:69)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 11:39:17 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/ORM/eclipse/workspace/Ormus Server/ormus.mv.db [1.4.188/7]" [50000-188]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2434)
	at org.h2.engine.Database.open(Database.java:671)
	at org.h2.engine.Database.openDatabase(Database.java:268)
	at org.h2.engine.Database.<init>(Database.java:262)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.getPlayer(PlayerDAO.java:19)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:69)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/ORM/eclipse/workspace/Ormus Server/ormus.mv.db [1.4.188/7]" [50000-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/ORM/eclipse/workspace/Ormus Server/ormus.mv.db [1.4.188/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 22 more
2015-11-24 11:52:23 jdbc[11]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.getPlayer(PlayerDAO.java:19)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:74)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:91)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 14:09:13 jdbc[10]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:526)
	at com.ormus.server.entity.World.run(World.java:100)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-24 14:12:20 jdbc[14]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.savePlayer(PlayerDAO.java:120)
	at com.ormus.server.entity.World.update(World.java:214)
	at com.ormus.server.entity.World.run(World.java:100)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 14:32:21 jdbc[17]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.savePlayer(PlayerDAO.java:120)
	at com.ormus.server.entity.World.update(World.java:214)
	at com.ormus.server.entity.World.run(World.java:100)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 18:44:34 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.savePlayer(PlayerDAO.java:120)
	at com.ormus.server.entity.World.update(World.java:564)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 19:03:04 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.savePlayer(PlayerDAO.java:120)
	at com.ormus.server.entity.World.update(World.java:215)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 19:06:49 jdbc[19]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:49)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:95)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-24 21:38:19 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.playerExists(PlayerDAO.java:67)
	at com.ormus.server.networking.AuthenticationManager$1.onMessageReceived(AuthenticationManager.java:49)
	at com.ormus.shared.networking.Connection$InputThread.run(Connection.java:95)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-25 08:42:59 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:543)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-25 14:53:48 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:543)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-25 15:00:29 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:545)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-25 16:42:04 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:545)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-25 17:40:49 jdbc[9]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:545)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-25 18:05:41 jdbc[9]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:545)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
2015-11-25 18:31:58 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.model.dao.PlayerDAO.savePlayer(PlayerDAO.java:120)
	at com.ormus.server.entity.World.update(World.java:215)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2015-11-25 19:35:58 jdbc[9]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.ormus.server.model.dao.BaseDAO.getConnection(BaseDAO.java:21)
	at com.ormus.server.entity.World.update(World.java:545)
	at com.ormus.server.entity.World.run(World.java:101)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 12 more
